### ISSUE TITLE
RMSProp documentation is confusing

### ISSUE THREAD

**Issue by aidan-fitz** [pytorch/pytorch#54368]
## 📚 Documentation

The documentation on the parameters for `torch.optim.RMSprop` is vague and seems to contradict itself. I couldn't tell what `alpha` means without looking at the code (it's the exponential discount factor for the second moment running average, which Keras calls `rho`). On the other hand, the sentence
> The effective learning rate is thus α/(sqrt(v) + ε) where α is the scheduled learning rate and v is the weighted moving average of the squared gradient.

uses alpha to mean the learning rate, which corresponds to `lr`. The documentation also does not explain how the second moment running average is computed, which would help explain how to use the discount rate `alpha`.

**Comment by heitorschueroff** [comment]
We would accept a PR that improves the existing documentation.
